.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,20 @@
-#### What does this PR do?
-#### Any background context you want to provide?
-#### What ticket does this PR close?
-Connected to [relevant GitHub issues, eg #76]
-#### Where should the reviewer start?
-#### How should this be manually tested?
-#### Screenshots (if appropriate)
-#### Has the Version and Changelog been updated?
-#### Questions:
-> Does this work have automated integration and unit tests?
+### What does this PR do?
+- _What's changed? Why were these changes made?_
+- _How should the reviewer approach this PR, especially if manual tests are required?_
+- _Are there relevant screenshots you can add to the PR description?_
 
-> Can we make a blog post, video, or animated GIF of this?
+### What ticket does this PR close?
+Connected to #[relevant GitHub issues, eg 76]
 
-> Has this change been documented (Readme, docs, etc.)?
+### Checklists
 
-> Does the knowledge base need an update?
+#### Change log
+- [ ] The CHANGELOG has been updated, or
+- [ ] This PR does not include user-facing changes and doesn't require a CHANGELOG update
+
+#### Test coverage
+- [ ] This PR includes new unit and integration tests to go with the code changes
```

Comment:
Would it be worth adding the negative option to check here as well? E.g.
```
 - [ ] No new or updated tests are required
```

This  would help distinguish between forgotten tests versus the author doesn't believe any new tests are are needed.

Reply:
I can add it, and considered it. I suppose that in leaving it without the negative I intended to imply that the negative case was not a thing, but of course it sometimes is (not least for my own work in this repo, which doesn't usually involve changing testable code).

I'll add it.

Reply:
```suggestion
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [ ] The changes in this PR do not require tests
```

Reply:
@micahlee let me know how that looks

Reply:
Looks good to me, thanks!